src/main/java/com/atlassian/jira/cloud/jenkins/common/client/JenkinsAppApi.java

```diff
@@ -62,7 +63,11 @@ protected ResponseEntity sendRequestAsJwt(
             final Class<ResponseEntity> responseClass)
             throws ApiUpdateFailedException {
         try {
-            final String requestPayload = wrapInJwt(jenkinsAppRequest, secret);
+            final String requestPayload =
+                    wrapInJwt(
+                            jenkinsAppRequest,
+                            secret,
+                            Date.from(Instant.now().plusSeconds(JWT_EXPIRY_SECONDS)));
```

Comment:
Allow to pass in an expiry date, because I wanted to generate expired and non-expired JWTs from tests